src/ts/util/cssHelper.ts

```diff
@@ -12,7 +12,7 @@ const viewMatchMap = {
   time: timeGetViewID
 };
 
-export function classname(str = ''): string {
+export function cls(str = ''): string {
```

Comment:
The `cls` is too short
It does not convey meaning well.
`getPrefixClassName` is too long ...?

Reply:
too difficult naming this method. It returns not prefix, but whole class name. So I think `getClassName` is proper. 
@seonim-ryu Come Here!!! LOL

Reply:
`cls` is confirmed by @dongsik-yoo, @jungeun-cho and @seonim-ryu.